vtex/billing-app
# Improve error messages from Billing malfunction

#### What is the purpose of this pull request?
Improve error messages from Billing malfunction

#### What problem is this solving?
When there is a problem with an account's Billing module, no useful error message is displayed in the toolbelt or admin, often caused by an error 500. This change tries to identify such errors and create a message that can be used to inform the users that their account has a problem with its Billing module.

#### How should this be manually tested?
1. Clone/pull this repo/branch and link in your workspace in the `vtexcs` account
2. Try to install an app with billing options, such as `vtex.masterpass`
3. Check the error message that follows

#### Screenshots or example usage

#### Types of changes
